### DISCUSSION TITLE
Multiple labels for one order id

### DISCUSSION THREAD

**Discussion by Dubn** [amzn/selling-partner-api-models#3376] [General]
Hello,

I'm reaching out to see if anyone has had any success with getting more then one shipment labels for one order id from the API  /mfn/v0/shipments?

I use /mfn/v0/shipments to get shipment labels and I have such a problem that I only get one order id for which I want to get more than one label, but there is no field specifying the number of parcels anywhere and only one parcel can be defined. Sending a request with an order id for each package also does not work, because at second request we get an error(the error doesn't even describe the problem with the second use of the order id, it just gives message and guid). Do you guys now how can I get more than one label for one order id?

**Comment by Bithellio** [comment]
I now this was posted a long time ago but is there a solution to this, ive come accross the same issue. we are dispatching large quantities of items and when an order reaches a certain size its not really feasable to cram it into a single box.  ( plus all carriers have weight limits for a single parcel ) 

i need to be able to state a shipment is x number of packages so i need x number of labels
